docs/commands/build-docs.md

```diff
@@ -0,0 +1,113 @@
+# `build-docs`
+
+## Introduction
+
+The `build-docs` command builds Redoc into a zero-dependency HTML file.
+
+## Usage
+
+```bash
+redocly build-docs <api>
+redocly build-docs <api> --output=custom.html
+redocly build-docs <api> --features.openapi.theme.colors.primary.main=orange
+redocly build-docs <api> -t custom.hbs
```

Comment:
`-t` isn't documented anywhere? Are there other option shortcuts?

Reply:
```suggestion
redocly build-docs <api> --template custom.hbs
```
Maybe I would recommend this for now.

Reply:
I forgot to add the aliases to the documentation.
Added. We can use both cases.
I am not sure about the best approach.